servers/rendering/rendering_device.cpp

```diff
@@ -211,7 +216,56 @@ Vector<uint8_t> RenderingDevice::shader_compile_spirv_from_source(ShaderStage p_
 
 	ERR_FAIL_NULL_V(compile_to_spirv_function, Vector<uint8_t>());
 
-	return compile_to_spirv_function(p_stage, p_source_code, p_language, r_error, this);
+	// Check for build in includes
+	const String include = "#include \"";
```

Comment:
Same as above.

Additionally, this could probably also be moved to a different function like "shader_run_preprocessor"

Reply:
There wasn't a really good place to place that so I moved the code into `ShaderIncludeDB::parse_include_files`.